### DISCUSSION TITLE
Community meeting August 20th 2021 (v1.39)

### DISCUSSION THREAD

**Discussion by lucasponce** [kiali/kiali#4282] [Community meetings]
Hello community,

Next Friday we will continue with the Kiali community meetings with the following format:

🗒️ **Agenda**

1. _Kiali-wide announcements_: A place to announce Kiali news
2. _Sprint / release demo_: Space to showcase the work introduced in the next release
3. _Submitted topics_: Place your questions, topics, thoughts, agenda items in the discussion below and we'll approach them.
4. _Open Q&A_: Space for remaining topics, questions, thoughts not covered in this discussion.

🎤 **Add a topic to the agenda**

Place a comment in this discussion with the topic you would like the community (contributors, maintainers, users) discuss during the meeting. In the _Submitted topics_ section we'll go through all the topics and will open the mics to approach it.
We encourage you to join the meeting in order to get a complete answer and get as far as we can.

ℹ️ **Community meeting info**

:calendar: Friday August 20th, 2021
:clock1030: 10am (EDT) / 4pm (CEST) / 10pm (HKT)
:computer: https://bluejeans.com/7625852117

**Comment by xeviknal** [comment]
One topic to the agenda:

@linjmeyer will do a ~15min demo about managing a service mesh the gitops way. Thanks! :tada:

**Comment by xeviknal** [comment]
I remember someone wanted to provide some links to the meeting attendants. What do you think of sharing them in here?
I start with these ones:
- https://iter8.tools/0.7/ - Tool for progressing rollouts
- https://kiali.io/documentation/v1.38/runtimes-monitoring/ - custom dashboards and runtime metrics in Kiali

**Comment by leandroberetta** [comment]
@xeviknal good idea, I want to share the following links regarding certificate management in Istio:

* https://istio.io/latest/docs/tasks/security/cert-management/plugin-ca-cert/
* https://istio.io/latest/docs/tasks/security/cert-management/dns-cert/

**Comment by linjmeyer** [comment]
Thanks again all for having me, looking forward to deep diving Kiali and getting more of the features setup!

Links from the demo:

* https://github.com/linjmeyer/istio-demo (demo files, spinnaker operator, etc)
* https://github.com/redboxllc/scuttle Scuttle, for handling Istio sidecars in Jobs/CronJobs
* https://github.com/foremast/foremast Foremast, a Spinnaker pipeline and infrastructure configuration and templating tool
